packages/fuels-code-gen/src/utils/source.rs

```diff
@@ -71,6 +71,14 @@ impl Source {
             Source::String(abi) => Ok(abi.clone()),
         }
     }
+
+    /// If the abigen is given a path, will return it here.
```

Comment:
IMO the comment is not necessary. If we still want to keep the comment, maybe we say `if the source is a path, return it` rather than `if the abigen is given a path...`